src/action-intent-request.js

```diff
@@ -0,0 +1,67 @@
+import {
+  buildSpeechResponse,
+  ResponseModel
+} from './response-model';
+
+import {
+  getRoundQuestions
+} from './questionStorage';
+
+let model = require('./response-model'),
+    utilities = require('./utilities'),
+    ActionLaunchRequest = require('./action-launch-request'),
+    client = require('./api-client');
```

Comment:
@steveortiz  fixed here fb8cff278e58cce257bac97e9dbf3eb452d1ce71.
Not sure if you'd like me to do the same for the rest of the variables, utilities and so forth, but they aren't explicitly exported from the files with those names so I think I'd have to do a bit more refactoring for that